setup.py

```diff
@@ -290,6 +310,7 @@ def run(self):
           python_requires='>=3.6',
           cmdclass={'clean': Clean,
                     'develop': Develop,
+                    'developexternal': DevelopExternal,
```

Comment:
I'm ok with having a new command, but since that you already defined a custom `develop` command, what about instead of using another command, if we had the `--external` option to the develop command? Have you considered that?

Reply:
@beraldoleal I think that's going to make the develop command too complicated for little gain. I also find `--external` might be confusing. The name of the command should something like develop-external-plugins to make obvious you don't need it unless you're developing an external plugin.
A possible solution here is that if you have defined `AVOCADO_EXTERNAL_PLUGINS_PATH` then when you use the develop command also will install the external plugins. This would entirely remove the command. It would also reinstall everything else, but it's a quick action so it doesn't seem like a big deal. What do you think?

Reply:
I think this approach is more prune to errors, since devs can have the env variable defined but don't remember, so IMO it is not so explicit.

I also don't think `--external` will make things complicated if they are documented. Take `python setup.py develop --help` and see all the options for develop for instance. We are just adding a new one. In fact, maybe we should respect `--no-deps` there, not sure.

But as I said, I'm ok with this approach as well, so I will let you choose. ;)